src/connectors/breadcrumb/__tests__/connectBreadcrumb-test.ts

```diff
@@ -579,15 +607,19 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/breadcrumb/
       ],
     });
 
-    const config = widget.getWidgetSearchParameters(new SearchParameters());
-    const helper = jsHelper({}, '', config);
+    const config = widget.getWidgetSearchParameters!(new SearchParameters(), {
+      uiState: {},
+    });
+    const helper = jsHelper(createSearchClient(), '', config);
     helper.search = jest.fn();
 
-    widget.init({
-      helper,
-      state: helper.state,
-      createURL: state => state,
-    });
+    widget.init!(
+      createInitOptions({
+        helper,
+        state: helper.state,
+        createURL: state => (state as unknown) as string,
```

Comment:
You can return `JSON.stringify(state)` to keep type safety (in other places too).